### ISSUE TITLE
Generate Note labels for ASIDE

### ISSUE THREAD

**Issue by fantasai** [speced/bikeshed#2562]
The IndexedDB spec is using `<aside class=note>` and none of these green boxes are getting the "Note:" label, but probably should.

**Comment by tabatkins** [comment]
They probably shouldn't be using `aside`, no? It's inline text.

**Comment by inexorabletash** [comment]
Huh, good point. Now where did I pick that up from...

**Comment by inexorabletash** [comment]
... but replacing `<aside class=note>` with `<div class=note>` doesn't provide the "Note:" label either. Source is https://github.com/w3c/IndexedDB/blob/main/index.bs (I haven't landed the s/aside/div/ change yet)... which stylesheet does the "Note:" label come from?

**Comment by inexorabletash** [comment]
Ah yes, Bikeshed autocreates it for `note: ....` markdown, not via styles.

**Comment by tabatkins** [comment]
Yeah, it's generated by the markdown form; if you're doing your own markup you're responsible for your own text, as it's an escape hatch.

**Comment by fantasai** [comment]
@tabatkins I suspect we've got a bunch of places where we're expecting the label to be generated, just like for .example. In fact the documentation seems to imply this will be the case:

https://speced.github.io/bikeshed/#notes-etc

So perhaps it makes sense to align Bikeshed with those expectations and spit out the "Note:" header on class="note" without an existing header.

**Comment by tabatkins** [comment]
Hm, the docs do pretty explicitly say that text will be added, yeah. Wonder if I used to do that and changed, or just wrote the wrong text? Anyway, I can probably change this, with a check that the text doesn't already start with "note".
